bookwyrm/templates/guided_tour/search.html

```diff
@@ -0,0 +1,171 @@
+{% load i18n %}
+
+<script>
+
+    let localResult = document.querySelector(".local-book-search-result");
+    let remoteResult = document.querySelector(".remote-book-search-result");
+    let otherCatalogues = document.querySelector("#tour-load-from-other-catalogues");
+    let manuallyAdd = document.querySelector("#tour-manually-add-book");
+    const tour = new Shepherd.Tour({
+        exitOnEsc: true,
+    });
+
+    console.log("remote", remoteResult)
+
+    if (remoteResult) {
+        tour.addStep(
+        {
+            text: "{% trans 'If the book you are looking for is available on a remote catalogue such as Open Library, click on' %}\
+            <code>\
+            {% trans 'Import book' %}\
+            </code>.",
+            title: "{% trans 'Searching' %}",
+            attachTo: {
+                element: "#tour-remote-search-result",
+                on: "top",
+            },
+            buttons: [
+                {
+                    action() {
+                        disableGuidedTour(csrf_token);
+                        return this.complete();
+                    },
+                    secondary: true,
+                    text: "{% trans 'End Tour' %}",
+                    classes: "is-danger guided-tour-cancel-button",
+                },
+                {
+                    action() {
+                        return this.next();
+                    },
+                    text: "{% trans 'Next' %}",
+                },
+            ],
+        });
+    } else if (localResult) {
+    tour.addStep(
+        {
+            text: "{% trans 'If the book you are looking for is already on this server, you can click on the title to go to the book\'s page.' %}",
```

Comment:
From an internal consistency perspective, `instance` is more correct than `server`. But I wonder if there's a way to phrase this that doesn't use it at all? I'm trying to avoid making people have to learn jargon....

Reply:
Agreed. In this particular case it does matter (I think?). Will have a think.

Reply:
Yeah, it's hard to explain books being available but not currently on your instance without talking about that explicitly...